### DISCUSSION TITLE
support of ICC profiles v2

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#5060] [Help]
Hello,
is the support of ICC profiles v2 will be there for a long time please ?
Thank you
Best regards

**Comment by snibgo** [comment]
IM uses the LCMS library to process ICC profiles. So I think your question should be addressed to LCMS, not IM.

**Comment by SylvainArd** [comment]
OK thank you have you their URL please ?
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers

**Reply by fmw42** [reply]
Search Google for LCMS. You should find their URL.

**Comment by snibgo** [comment]
I don't have a URL for LCMS.

**Comment by SylvainArd** [comment]
Ok thank you
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers


Le mer. 20 avr. 2022 à 18:42, Fred Weinhaus ***@***.***> a
écrit :

**Reply by blue-j** [reply]
ICC profile v2 is the older specification, although they have updated it to 2.4 recently.  The very latest is v5 (iccMAX) but v4 is very common.  Check the [ICC site](https://color.org/icc_specs2.xalter) for details.

[Little CMS (LCMS)](https://www.littlecms.com/) is what provides color management operations to ImageMagick.  It most certainly supports v2 ICC profiles, in addition to v4.  Perhaps there is a question behind your question?  What are you trying to do?  What version of ImageMagick are you using?  Did you make from source, or...?

- J
